npms/skale-observer/observer.mjs

```diff
@@ -929,6 +929,8 @@ export function mergeSChainsArrayFromTo( arrSrc, arrDst, arrNew, arrOld, opts )
 }
 
 let gArrSChainsCached = [];
+const gArrCacheHistory = [];
+let nMaxSizeOfArrCacheHistory = 20;
 
 export async function cacheSChains( strChainNameConnectedTo, addressFrom, opts ) {
```

Comment:
Propose to remove addressFrom parameter from cacheSchains - it is unnecessary parameter